### ISSUE TITLE
Update to v10.10.0

### ISSUE THREAD

**Issue by Thefrank** [Thefrank/jellyfin-server-freebsd#87]
Build:

- [x] Build Jellyfin-Server for FreeBSD.13-x64
- [x] Build Jellyfin-Server for FreeBSD.14-x64
- [ ] Build Jellyfin-Web for FreeBSD.13-x64
 - `Error: Embedded Dart Sass couldn't find the embedded compiler executable. Please make sure the optional dependency sass-embedded-freebsd-x64 is installed in node_modules.`
  - F
- [ ] Build Jellyfin-Web for FreeBSD.14-x64
 - See above

Train of thought notes might delete later:

---

Changes in the `sass` dependencies seem to have caused this. I guess there is no longer a Dart2JS fallback? `node-sass` is EOL https://github.com/sass/node-sass/issues/3424 "Projects that still use it should move onto [Dart Sass](https://sass-lang.com/dart-sass)." OSes that are not supported by Dart get left out? 
Dart is a Google project. RIP

Some light reading:
In the past 11 years the dart-lang team has not been given the resources to support FreeBSD.
https://github.com/dart-lang/sdk/issues/10260 and https://github.com/dart-lang/sdk/issues/52374 

Others:
https://github.com/nanoc/nanoc/issues/1710
https://github.com/sass-contrib/sass-embedded-host-ruby/issues/227
https://github.com/sass/dart-sass/issues/1173
https://github.com/jekyll/jekyll/issues/9462
https://github.com/owen2345/camaleon-cms/issues/1098

Progress?:

https://github.com/sass/dart-sass/issues/2325 with https://github.com/sass/dart-sass/pull/2413 and https://github.com/sass/embedded-host-node/pull/344

**Comment by nex3** [comment]
> I guess there is no longer a Dart2JS fallback?

There definitely is a pure JS distribution of Dart Sass which should work totally fine on FreeBSD. It's available as the `sass` package on npm.

**Comment by Thefrank** [comment]
Thanks for the heads up! I will see what I can do about working around the requirement for an embedded compiler when I get some time later in the week

**Comment by bapt** [comment]
my search engine juste made me encounter this issue, for your information I have updated the FreeBSD port to 10.10.1 and added a port for libskiasharp (built in the ports tree) to workaround sass-embeded issue, I just removed from the package*.json all reference to sass-embeded and it fallsback on using sass

**Comment by Thefrank** [comment]
@bapt Thank you! Sorry about not replying sooner.

Everyone else:
This is now in ports for people missing it here. It should be on `lastest` soonish https://www.freshports.org/multimedia/jellyfin/
Those on `quarterly` will not get this until the next quarterly. There were no security fixes.

This repo is now redundant, in a good way, but I will be shutting it down in the next few days.

**Comment by bapt** [comment]
> @bapt Thank you! Sorry about not replying sooner.
> 
> Everyone else:
> This is now in ports for people missing it here. It should be on `lastest` soonish https://www.freshports.org/multimedia/jellyfin/
> Those on `quarterly` will not get this until the next quarterly. There were no security fixes.
> 
> This repo is now redundant, in a good way, but I will be shutting it down in the next few days.

given you have been the first to work on this if you want maintainership or want to help working on the ports you are more welcome! thanks a lot for all your work!!!!

**Comment by Thefrank** [comment]
@bapt
TL;DR: You/others maintaining the port is fine. I will stick a big "Use pkg/ports to install jellyfin" at the top to reduce confusion.

Full:
Oh! I am not worried about others maintaining the port. My concern was more over any confusion by users on where to find the binaries and where to get support. Locking/removing this repo should(?) help resolve that if you want to go that direction.

"jellyfin freebsd", on google, shows my repo second, under Jellyfin's official site; the order is reversed on bing. Freshports is at the bottom of the first page for google and does not make the first page for bing. 

I had kept this repo open as an alternative for users that wanted a jellyfin binary sooner than what was provided in ports. 
I also made a few scripts to run in azure pipelines that would run tests and update the `Makefile` and others in the ports folder for those that wanted to use `poudriere` to build a package. Nice to have, but not really "port maintainer" rigorous (e.g., it broke when moving to `sass-embeded`, `libSkiaSharp` was a binary as I did not have time to make a correct port for it). I feel I do not have the time to maintain a port, but I do leave my process where I can (comments were removed later to keep the file clean https://github.com/Thefrank/jellyfin-server-freebsd/commit/e0d119745a97f67c47148ec267798197364203c5)

I also don't mind continuing jellyfin support here. People gravitate here for support anyways even after a port arrived on FreeBSD. 
I will fix the readme to reflect the correct location (`pkg install` or ports) for acquiring the package.
